catalog_reader/scripts/apps_hashes.py

```diff
@@ -67,22 +68,37 @@ def update_catalog_hashes(catalog_path: str) -> None:
             catalog_base_lib_dir_path = os.path.join(library_dir.as_posix(), lib_version)
             shutil.copytree(catalog_base_lib_dir_path, app_base_lib_dir.as_posix())
 
+            old_version = app_config["version"]
+            if bump_type and app_config['lib_version_hash'] != hashes[lib_version]:
+                if not is_valid_bump_type(bump_type):
+                    print(
+                        f'[\033[91mERROR\x1B[0m]\tInvalid bump type {bump_type!r}, '
+                        f'skipping version bumping for {app_dir.name!r} in {train_dir.name}'
+                    )
+                else:
+                    app_config["version"] = bump_version(old_version, bump_type)
+                    rename_versioned_dir(old_version, app_config['version'], train_dir.name, app_dir)
+
             app_config['lib_version_hash'] = hashes[lib_version]
             with open(str(app_metadata_file), 'w') as f:
                 f.write(yaml.safe_dump(app_config))
 
-            print(f'[\033[92mOK\x1B[0m]\tUpdated library hash for {app_dir.name!r} in {train_dir.name}')
+            print(
+                f'[\033[92mOK\x1B[0m]\tUpdated library hash for {app_dir.name!r} in {train_dir.name}'
+                f' and bumped version from {old_version!r} to {app_config["version"]!r}' if bump_type else ''
+            )
 
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('--path', help='Specify path of TrueNAS catalog')
+    parser.add_argument('--bump', default=None, help='Bump type for app that the hash was updated')
```

Comment:
`add_argument` supports boolean types so instead of instantiating this to a `NoneType`, you can do something like so:
```
parser.add_argument('--bump', type=bool, default=False, action='store_true', help='...
```
This makes is so that if someone passes the `--bump` argument, it will be set to `True` otherwise, it'll default to `False`

Reply:
Oh its not a toggle, its an optional string (major, minor, patch). I should probably change the name.

Reply:
Oh, yeah definitely :sweat_smile: